lnst/RecipeCommon/Perf/Recipe.py

```diff
@@ -93,6 +94,20 @@ def add_measurement_results(
         )
         self._aggregated_results[measurement] = aggregated_results
 
+    """
+        Function returns end timestamp of warmup period and start of warm down period.
+        That results to slice measurement just for "interesting" part not including 
+        warm up and warm down periods.
+    """
+    def _get_real_flow_times(self, flows: List[BaseMeasurementResults]):
```

Comment:
Hmm, well, this method returns the warmup end and warmdown start, so not sure to what should i rename it. It just returns the timestamps of flow between warmup and warmdown

Reply:
The docstring states it returns __timestamps__, so I'd not call it `_get_.._times`. `flow_times` sounds like it should return the duration of flows. The name should be as much descpritive as possible. So if the docstring states it returns two timestamps, then it should be named accordingly.

Reply:
Additionally, I think the name should not include Flow at all. The type hint `flows: List(BaseMeasurementResults])` is confusing. I'd rename the flows to results.

Reply:
if anything i'd include "measurement" instead of "flow" as that's more accurate to what the timestamps work with, it's also what the type hints suggest.